src/test/java/com/contrastsecurity/maven/plugin/sdkx/GsonFactoryTest.java

```diff
@@ -19,14 +21,59 @@ void parse_iso8601_strings_to_local_date_time() {
     final Gson gson = GsonFactory.create();
 
     // WHEN deserialize a type containing a LocalDateTime
-    final Obj obj = gson.fromJson("{\"time\": \"1955-11-12T22:04:00\"}", Obj.class);
+    final LocalDateTimeObj obj =
+        gson.fromJson("{\"time\": \"1955-11-12T22:04:00\"}", LocalDateTimeObj.class);
 
     final LocalDateTime expected = LocalDateTime.of(1955, 11, 12, 22, 4, 0);
     assertThat(obj.time).isEqualTo(expected);
   }
 
+  @Test
+  void parse_null_to_local_date_time() {
+    // GIVEN a new GSON instance that should handle java.time.LocalDateTime
+    final Gson gson = GsonFactory.create();
+
+    // WHEN deserialize a type containing a LocalDateTime that is null
+    final LocalDateTimeObj obj = gson.fromJson("{\"time\": null}", LocalDateTimeObj.class);
```

Comment:
Good catch with this. Didn’t realize that the handlers always need the deal with potential null values.